.ci.yaml

```diff
@@ -529,6 +529,90 @@ targets:
           {"dependency": "android_virtual_device", "version": "33"}
         ]
 
+  - name: Linux_android android_platform_tests_legacy_api_shard_1 master
+    recipe: packages/packages
+    timeout: 60
+    bringup: true # New target
+    properties:
+      target_file: android_legacy_emulator_tests.yaml
+      channel: master
+      version_file: flutter_master.version
+      package_sharding: "--shardIndex 0 --shardCount 6"
+      dependencies: >-
+        [
+          {"dependency": "android_virtual_device", "version": "22"}
+        ]
+
+  - name: Linux_android android_platform_tests_legacy_api_shard_2 master
+    recipe: packages/packages
+    timeout: 60
+    bringup: true # New target
+    properties:
+      target_file: android_legacy_emulator_tests.yaml
+      channel: master
+      version_file: flutter_master.version
+      package_sharding: "--shardIndex 1 --shardCount 6"
+      dependencies: >-
+        [
+          {"dependency": "android_virtual_device", "version": "22"}
+        ]
+
+  - name: Linux_android android_platform_tests_legacy_api_shard_3 master
+    recipe: packages/packages
+    timeout: 60
+    bringup: true # New target
+    properties:
+      target_file: android_legacy_emulator_tests.yaml
+      channel: master
+      version_file: flutter_master.version
+      package_sharding: "--shardIndex 2 --shardCount 6"
+      dependencies: >-
+        [
+          {"dependency": "android_virtual_device", "version": "22"}
+        ]
+
+  - name: Linux_android android_platform_tests_legacy_api_shard_4 master
+    recipe: packages/packages
+    timeout: 60
+    bringup: true # New target
+    properties:
+      target_file: android_legacy_emulator_tests.yaml
+      channel: master
+      version_file: flutter_master.version
+      package_sharding: "--shardIndex 3 --shardCount 6"
+      dependencies: >-
+        [
+          {"dependency": "android_virtual_device", "version": "22"}
```

Comment:
Out of scope for this pr but I wish we could use variables for these shards since they cant actually have different dependencies.

Reply:
I could make a new config for legacy emulator Android tests, but without a way to have one `platform_properties` entry inherit from another one that sort of just moves the problem (since I'd have to instead duplicate all the core Android dependencies, which can't really be different either). Would you prefer that? I can see an argument for it being less error-prone.

Reply:
Since this is a whole new set of shards with configs, it seems in scope to have it start the way you'd prefer it; I'll leave the existing ones alone for now though (especially so I don't mess up Camille's WIP PR).

Reply:
Let me know what you think of this updated version :)